lib/metadata/find-removed-reviewers.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+var github = require('../github');
+
+function findRemovedReviewers(number) {
+    return github.get("/repos/:owner/:repo/issues/:number/events", {number: number})
+        .then(function(response) {
+            if (!response /* TODO or something */) {
```

Comment:
This isn't very actionable. Can you be more specific about what you'd like to see done?

Reply:
I wasn't sure what shape response would have when there is an error, and haven't tested for that.